Kickstarter-iOS/Views/Controllers/CommentRepliesViewController.swift

```diff
@@ -33,7 +33,7 @@ final class CommentRepliesViewController: UITableViewController {
     super.viewDidLoad()
 
     // TODO: Internationalize
-    self.navigationItem.title = "Replies"
+    self.navigationItem.title = localizedString(key: "", defaultValue: "Replies")
```

Comment:
Should be ` localizedString(key: "Replies", defaultValue: "Replies")`

Reply:
Ah yes good call, wasn't sure about the key :) Will update now.